### PR TITLE
terraform --print-to-file handle output for multiple accounts

### PR DESCRIPTION
**Pull request by geoberle**
the --print-to-file flag of terraform integrations results in the contents managed by terrascript to be dumped to a file. currently, only the contents of the last handled account is present in the file, because the dump loop keeps overwriting the same file over and over again.

this change makes sure that the file will contain the tf information from all accounts, delimited by `##### account name #####`

Ref: https://issues.redhat.com/browse/APPSRE-4421
Signed-off-by: Gerd Oberlechner <goberlec@redhat.com>

### CODE REVIEW

File: reconcile/utils/terrascript_client.py [hunk 1]
```diff
@@ -2992,13 +2992,18 @@ def dump(self, print_to_file=None, existing_dirs=None):
...
+        if os.path.isfile(print_to_file):
+            os.remove(print_to_file)
+
         for name, ts in self.tss.items():
             if print_to_file:
-                if is_file_in_git_repo(print_to_file):
-                    raise PrintToFileInGitRepositoryError(print_to_file)
-                with open(print_to_file, 'w') as f:
+                with open(print_to_file, 'a') as f:
                     f.write('##### {} #####\n'.format(name))
```
[the remark this answers is not in the document]
Comment (fishi0x01) [reply reconcile/utils/terrascript_client.py:3004]: Oh sorry - thats actually not part of your change.

[the remark this answers is not in the document]
Comment (geoberle) [reply reconcile/utils/terrascript_client.py]: while we are at it :) fixed in 51d3c23

**Review by fishi0x01 — APPROVED** [review]
LGTM

**Comment by maorfr** [comment]
w00t w00t!

**Pull request merged**